src/resources/js/frontend-ticket-form.js

```diff
@@ -109,8 +109,9 @@ var tribe_tickets_ticket_form = {};
 			}
 
 			if ( 'capped' === ticket.mode ) {
-				remaining = ticket.cap - $( '[data-product-id=' + ticket_id + ']' ).find( 'input' ).val();
-				$tickets_lists.find( '.available-stock[data-product-id=' + ticket_id + ']').html( remaining );
+				var effective_cap = Math.min( remaining, ticket.cap );
+				var remaining_capped_stock = effective_cap - $( '[data-product-id=' + ticket_id + ']' ).find( 'input' ).val();
```

Comment:
With some providers, this _can_ find a hidden input value that contains an id attributed. I'd suggest the following:

```js
$( '[data-product-id=' + ticket_id + ']' ).find( 'input:not([type="hidden"])' ).val();
```